frontend/app/src/main/java/dk/scheduling/schedulingfrontend/pages/CreateTaskPage.kt

```diff
@@ -31,126 +32,137 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.text.input.KeyboardType
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import dk.scheduling.schedulingfrontend.api.protocol.Device
 import dk.scheduling.schedulingfrontend.components.DateRange
 import dk.scheduling.schedulingfrontend.components.FilledButton
+import dk.scheduling.schedulingfrontend.components.Loading
 import dk.scheduling.schedulingfrontend.components.StandardDateRangePicker
 import dk.scheduling.schedulingfrontend.components.StandardTimePickerDialog
 import dk.scheduling.schedulingfrontend.components.Title
 import dk.scheduling.schedulingfrontend.model.Duration
 import dk.scheduling.schedulingfrontend.model.TaskForm
+import dk.scheduling.schedulingfrontend.repositories.device.IDeviceRepository
 import dk.scheduling.schedulingfrontend.sharedcomponents.StandardDropDownMenu
 import dk.scheduling.schedulingfrontend.ui.theme.SchedulingFrontendTheme
+import testdata.DummyDeviceRepository
 
 @Composable
 fun CreateTaskPage(
     modifier: Modifier = Modifier,
-    handleSubmission: () -> Unit,
-    handleCancellation: () -> Unit,
+    deviceRepo: IDeviceRepository,
 ) {
-    Title(titleText = "Create Task", topMargin = 0.dp)
-
-    Column(
-        modifier =
-            modifier
-                .fillMaxSize()
-                .padding(20.dp),
-        horizontalAlignment = Alignment.CenterHorizontally,
-        verticalArrangement = Arrangement.SpaceBetween,
+    val loading =
+        remember { mutableStateOf(true) }
+    val devices =
+        remember { mutableStateOf(listOf<Device>()) }
+    Loading(
+        isLoading = loading.value,
+        setIsLoading = { loading.value = it },
```

Comment:
If accept the previous comment then change this to:
```suggestion
        isLoading = isLoading,
        setIsLoading = setIsLoading,
```